contracts/contracts/proxies/Proxies.sol

```diff
@@ -346,3 +346,10 @@ contract PoolBoostCentralRegistryProxy is
 contract MakerSSRStrategyProxy is InitializeGovernedUpgradeabilityProxy {
 
 }
+
+/**
+ * @notice MakerSSRStrategyProxy delegates calls to a Generalized4626Strategy implementation
```

Comment:
The Natspec should be changed to `OUSDCurveAMOProxy delegates calls to a OUSDCurveAMOStrategy implementation`